buildroot-external/board/raspberrypi/kernel.config

```diff
@@ -1,2 +1,3 @@
 # CONFIG_AUTOFS4_FS is not set
 # CONFIG_AUTOFS_FS is not set
+CONFIG_PANIC_TIMEOUT=5
```

Comment:
That would be better at https://github.com/home-assistant/operating-system/blob/dev/buildroot-external/kernel/hassos.config

Reply:
Thanks. Moved to central config.